cranelift/docs/ir.md

```diff
@@ -95,11 +95,13 @@ execution can never fall through to the next EBB without an explicit branch.
 
 A `.clif` file consists of a sequence of independent function definitions:
 
-.. productionlist::
-    function_list : { function }
-    function      : "function" function_name signature "{" preamble function_body "}"
-    preamble      : { preamble_decl }
-    function_body : { extended_basic_block }
+```
+productionlist::
```

Comment:
```suggestion
```

That was a `rst` annotation for the code block language, just like

````
```productionlist
````

for markdown.